language/polkavm/move-to-polka/tests/examples_basic_test.rs

```diff
@@ -124,12 +97,29 @@ pub fn test_tuple_implementation() -> anyhow::Result<()> {
 
 #[test]
 #[serial]
-pub fn test_multi_module_call() -> anyhow::Result<()> {
-    initialize_logger();
-    let build_options = BuildOptions::new("output/multi_module_call.polkavm")
-        .source("../examples/multi_module/sources/modules2.move")
-        .address_mapping("multi_module=0x7");
+pub fn test_multi_module_call2() -> anyhow::Result<()> {
+    let mut instance = build_instance(
+        "output/multi_module_call.polkavm",
+        "../examples/multi_module/sources/modules2.move",
+        vec!["multi_module=0x7"],
+    )?;
+
+    // Grab the function and call it.
+    info!("Calling into the guest program (high level):");
+    let result = instance
+        .call_typed_and_get_result::<u32, (u32, u32, u32)>(&mut (), "add_all", (10, 5, 5))
+        .map_err(|e| anyhow::anyhow!("{e:?}"))?;
+    assert_eq!(result, 20);
 
+    Ok(())
+}
+
+fn build_instance(output: &str, source: &str, mapping: Vec<&str>) -> anyhow::Result<Instance> {
+    initialize_logger();
```

Comment:
Nit: This feels misplaced considering the function name. But I can understand that it is easiest to place it here.

Reply:
yeah, maybe a better name for the function would be something like `init_basic`